code/game/machinery/computer/manager_camera.dm

```diff
@@ -1,6 +1,29 @@
 /obj/machinery/computer/camera_advanced/manager
 	name = "managerial camera console"
-	var/datum/action/innate/door_bolt/bolt_action = new
+	desc = "A computer used for remotely handling a facility."
+	icon_screen = "mechpad"
+	icon_keyboard = "generic_key"
+	var/datum/action/innate/door_bolt/bolt_action
+	var/datum/action/innate/healthheal/hpbullet
+	var/datum/action/innate/sanityheal/spbullet
+	var/datum/action/innate/redshield/rshield
+	var/datum/action/innate/whiteshield/wshield
+	var/datum/action/innate/blackshield/bshield
+	var/datum/action/innate/paleshield/pshield
+	var/datum/action/innate/slowdown/abnoslowdown
+	var/ammo = 0
+	var/maxAmmo = 5
```

Comment:
All of it. Yes: ALL of it can be reduced to 2 variables: Action to fire bullet, and currently loaded bullet type.
Please do so. If you don't exactly understand what I mean, I could do it myself.